src/app/technical-record/vehicle-summary/vehicle-summary.component.html

```diff
@@ -283,6 +304,36 @@ <h2 class="govuk-heading-m">Type Approval</h2>
     </dd>
     <dd class="govuk-summary-list__value"></dd>
   </div>
+
+  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType === 'psv'">
+    <dt class="govuk-summary-list__key">
+      COIF serial number
+    </dt>
+    <dd id="test-vehicleSummary-coifSerialNumber" class="govuk-summary-list__value">
+      {{ activeRecord.coifSerialNumber | DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType === 'psv'">
+    <dt class="govuk-summary-list__key">
+      COIF certifier name
+    </dt>
+    <dd id="test-vehicleSummary-coifCertifierName" class="govuk-summary-list__value">
+      {{ activeRecord.coifCertifierName | DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType === 'psv'">
+    <dt class="govuk-summary-list__key">
+      COIF date
+    </dt>
+    <dd id="test-vehicleSummary-coifDate" class="govuk-summary-list__value">
+      {{ activeRecord.coifDate | DefaultNullOrEmpty }}
+    </dd>
```

Comment:
should all these fields be inside`vtm-vehicle-summary-psv`

Reply:
they are unde type approval not the other fields